modules/api/src/main/java/org/eclipse/xpanse/api/controllers/ServicePolicyManageApi.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.api.controllers;
+
+import static org.eclipse.xpanse.modules.models.security.constant.RoleConstants.ROLE_ISV;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.annotation.Resource;
+import jakarta.validation.Valid;
+import java.util.List;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.xpanse.modules.models.policy.servicepolicy.ServicePolicy;
+import org.eclipse.xpanse.modules.models.policy.servicepolicy.ServicePolicyCreateRequest;
+import org.eclipse.xpanse.modules.models.policy.servicepolicy.ServicePolicyUpdateRequest;
+import org.eclipse.xpanse.modules.policy.policyman.PolicyManager;
+import org.eclipse.xpanse.modules.security.IdentityProviderManager;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+
+/**
+ * REST interface methods for service managing policies of the cloud service provider.
+ */
+@Slf4j
+@RestController
+@RequestMapping("/xpanse")
+@CrossOrigin
+@Secured({ROLE_ISV})
```

Comment:
just to keep it uniform with other controllers, we can add ADMIN_ROLE as well here.

Reply:
Updated as suggested.